src/SurveyService.Models.Dto/Responses/Question/QuestionResponse.cs

```diff
@@ -0,0 +1,21 @@
+using LT.DigitalOffice.SurveyService.Models.Dto.Models;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Responses.Question;
+
+public record QuestionResponse
+{
+  public Guid Id { get; set; }
+  public Guid? GroupId { get; set; }
+  public string Content { get; set; }
+  public DateTime? Deadline { get; set; }
+  public bool HasRealTimeResult { get; set; }
+  public bool IsAnonymous { get; set; }
+  public bool IsRevoteAvailable { get; set; }
+  public bool IsObligatory { get; set; }
+  public bool IsPrivate { get; set; }
+  public bool HasMultipleChoice { get; set; }
+  public bool HasCustomOptions { get; set; }
```

Comment:
Maybe you should put the question properties in the QuestionInfo model? Get Group also uses this model

Reply:
@AlfaRedDog not exactly the same: in Response there is IsActive; in Info - it shouldn't be (only active question are included); may be some more properties are unnecessary in Info (think about it)